mautic/mautic
# Fix unsubscribe form usage

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
In some cases unsubscribe form in emails make issues. This PR fixed it



#### Steps to test this PR:
2. Upload **Blank test** theme [blank-test.zip](https://github.com/mautic/mautic/files/6074396/blank-test.zip) - it's just email theme
3. Create form
4. Create email with **Blank test** theme and choose unsubscribe form
5. Send email
6. Try unsubscribe in email - you should see error
7. Enable in configuration Preferences center
8. Try unsubscribe in email again - you should see error
9. Create landing page as preferences center and assign this page to email created above
10. Try unsubscribe in email again - you should see error

After Patch all this situation should works properly

1. Load up [this PR](https://mautibox.com)
2. Repeat the steps above which should not cause an error
3. Check:

- disabled preferences center + unsubscribe form
- enabled preferences center + unsubscribe form
- enable preferences center + preferences center landing page + unsubscribe form




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9752"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
